common/record.js

```diff
@@ -0,0 +1,35 @@
+(function () {
+    'use strict';
+
+    angular.module('chaise.recordDisplay', [])
+
+    .directive('recordDisplay', ['ErrorService', function(ErrorService) {
+        return {
+            restrict: 'E',
+            transclude: true,
```

Comment:
I'm new to transclusion in Angular, so I might just be missing something, but why is transclusion necessary here?